bindings/ffi/RegorusFFI.g.cs

```diff
@@ -12,7 +12,7 @@ namespace RegorusFFI
 {
     internal static unsafe partial class API
     {
-        const string __DllName = "regorusc";
+        const string __DllName = "regorus_ffi";
```

Comment:
I don't know what changed this in my local system?

Reply:
Probably I missed updating this in my last PR; but it gets auto-generated anyways. Maybe I will delete it in another PR.